docs/ECO-METADATA.md

```diff
@@ -44,3 +44,48 @@ The schema also provides an API endpoint to avoid parsing json, however this end
 curl <http://169.254.169.254/eve/v1/external_ipv4>
 
 192.168.1.10
+
+## Location API endpoint
+
+Meta-data service allows applications to obtain geographic coordinates of the device,
+determined using Global Navigation Satellite Systems. This is available only if device
+has a supported LTE modem with an integrated GNSS receiver (e.g. Sierra Wireless EM7565).
+Standalone GPS receivers are currently not supported.
+
+However, by default EVE does not use the location service of the LTE modem and the location
+information is therefore not available.
+To enable location reporting for the device, the `wwan*` port (corresponding to the LTE modem)
+must be **configured for management** with **location tracking** enabled.
+In EVE API this is done by setting the field `NetworkConfig.wireless.cellularCfg.location_tracking`
+to `true`. For more details refer to [netconfig.proto](../api/proto/config/netconfig.proto).
+
+Provided that location tracking is enabled and the device has a good reception of GNSS signal,
+a JSON-formatted location information is made available to all applications on the
+`/eve/v1/location.json` endpoint.
+
+For example:
+
+```
```

Comment:
Add shell after the backqoutes to make markdownlint happy

Reply:
done